### PR TITLE
feat(docs) how to process graph data with labels

### PR DESCRIPTION
**Pull request by Elssky**

### CODE REVIEW

**Review by acezen — APPROVED** [review]

File: docs/libraries/cpp/getting-started.md [hunk 1]
```diff
@@ -277,6 +277,96 @@ is used to write the results to new generated data chunks.
...
+        arrow::io::default_io_context(),
+        input,
+        read_options,
+        parse_options,
+        convert_options).ValueOrDie();
+
+    std::shared_ptr<arrow::Table> table;
+    table = reader->Read().ValueOrDie();
+```
+Exporting data to disk is easier.s
```
Comment (lixueclaire) [inline docs/libraries/cpp/getting-started.md:314]: Exporting data to disk is easier.

File: docs/libraries/cpp/getting-started.md [hunk 2]
```diff
@@ -277,6 +277,96 @@ is used to write the results to new generated data chunks.
...
+``` cpp
+// write arrow table as chunk parquet
```
Comment (lixueclaire) [inline docs/libraries/cpp/getting-started.md:316]: Could you please format the code in the document,  particularly the alignment?

File: docs/libraries/cpp/getting-started.md [hunk 3]
```diff
@@ -277,6 +277,96 @@ is used to write the results to new generated data chunks.
...
+    std::cout << "id: " << it.id() << " ";
+    if (!label_result.has_error()) {
+      for (auto label : label_result.value()) {
+        std::cout << label << " ";
+      }
+    }
+    // ...
+  }
+
+   // query vertices based on a query result
```
Comment (lixueclaire) [inline docs/libraries/cpp/getting-started.md:355]: ditto

File: docs/libraries/cpp/getting-started.md (hunk 1 above)
Comment (Elssky) [reply docs/libraries/cpp/getting-started.md]: corrected

File: docs/libraries/cpp/getting-started.md (hunk 2 above)
Comment (Elssky) [reply docs/libraries/cpp/getting-started.md]: yes, corrected

**Review by lixueclaire — APPROVED** [review]
LGTM！

**Pull request merged**